src/ng-workspace.ts

```diff
@@ -59,6 +60,9 @@ export interface NgPublishToGitConfig {
 
 export interface PackageInfo {
   version: string;
+  scripts?: {
+    prePublishToGit: string;
```

Comment:
prePublishToGit: string;   -> prePublishToGit?: string;  De key is niet verplicht en kan dus ontbreken.